src/labels/CustomLabels.labels

```diff
@@ -2416,6 +2416,147 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUIDescription</shortDescription>
+        <value>
+            Connect the field you created on the NPSP Data Import object to the field on your target object in
+            the grid below. You can find fields by Field Label or the Field API Name.
+            Fields on the NPSP Data Import object must map to compatible field types on your target object.
+            When you select an NPSP Data Import field, only the compatible target fields will be available.
+        </value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUIFieldAPIName</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUIFieldAPIName</shortDescription>
+        <value>Field API Name</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUIFieldLabel</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUIFieldLabel</shortDescription>
+        <value>Field Label</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUILongDeployment</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUILongDeployment</shortDescription>
+        <value>Metadata deployment is taking longer than expected.</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUILongDeploymentLink</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUILongDeploymentLink</shortDescription>
+        <value>here</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUILongDeploymentMessage</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUILongDeploymentMessage</shortDescription>
+        <value>Your deployment will continue to process in the background. View the deployment status</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUINewFieldMapping</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUINewFieldMapping</shortDescription>
+        <value>Create New Field Mapping</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUINoFieldMappings</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUINoFieldMappings</shortDescription>
+        <value>There are no Field Mappings for this Mapping Group yet, </value>
```

Comment:
Please note "Mapping Group" term may change.

Note to self: trailing space seems to be ignored in code.